commercetools-models/src/test/java/io/sphere/sdk/products/PriceUtilsTest.java

```diff
@@ -0,0 +1,137 @@
+package io.sphere.sdk.products;
+
+import com.neovisionaries.i18n.CountryCode;
+import io.sphere.sdk.carts.TaxedItemPrice;
+import io.sphere.sdk.taxcategories.SubRate;
+import io.sphere.sdk.taxcategories.TaxRate;
+import io.sphere.sdk.utils.MoneyImpl;
+import org.junit.Test;
+
+import javax.annotation.Nonnull;
+import javax.annotation.Nullable;
+import javax.money.MonetaryAmount;
+import java.math.BigDecimal;
+import java.util.List;
+
+import static io.sphere.sdk.products.PriceUtils.*;
+import static io.sphere.sdk.test.SphereTestUtils.EUR;
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class PriceUtilsTest {
+
+    @Test
+    public void returnsZeroAmount() throws Exception {
+        final MonetaryAmount zeroAmount = zeroAmount(EUR);
+        assertThat(zeroAmount.getNumber().doubleValue()).isEqualTo(0);
+        assertThat(zeroAmount.getCurrency()).isEqualTo(EUR);
+    }
+
+    @Test
+    public void convertsToGross() throws Exception {
+        final MonetaryAmount netAmount = monetaryAmountOf(100);
```

Comment:
I considered that but the results could be a bit confusing and I forgot to add a comment here.

The problem is that the Moneta implementation of Money is not rounding (which is good actually because we want to keep the most precise value until the very end) but then some  prices might be returned as `10.33333333333...` which looks weird in the tests.

WDYT?